### DISCUSSION TITLE
Como Alterar Prazo de Expiraçao Script Criaçao de Usuario

### DISCUSSION THREAD

**Discussion by edmato** [bacen/pix-api#517] [Q&A]
Bom dia, tenho como atividade fazer inclusao de bancos no Redash, uma parte do fluxo do trabalho é gerar o Script no Git de Criaçao de Usuario e Senhas. 
É criado uma RFC com o link dos script e encaminhado para o time executor. 
Problema que depois de executado o script e finalizado RFC, o link do script nao expira, com isso todo mundo continua tendo acesso as senhas dos usuario á dados sensiveis, mesmo com o card fechado.

**Comment by rubenskuhl** [comment]
> Bom dia, tenho como atividade fazer inclusao de bancos no Redash, uma parte do fluxo do trabalho é gerar o Script no Git de Criaçao de Usuario e Senhas. É criado uma RFC com o link dos script e encaminhado para o time executor. Problema que depois de executado o script e finalizado RFC, o link do script nao expira, com isso todo mundo continua tendo acesso as senhas dos usuario á dados sensiveis, mesmo com o card fechado.

E o que isso tem a haver com API Pix ?

**Comment by edmato** [comment]
Bom dia Rubens tudo bom ...
Realmente de fato nao tem nada a ver ... irei encaminhar minha duvida para outro canal. 
Desde já agredeço sua "atençao".
Obrigado
